spec/internal/app/controllers/posts_controller.rb

```diff
@@ -11,6 +12,12 @@ class PostsController < Matterhorn::Base
 
   add_env :current_user
 
+  paginates_with Matterhorn::Paging::Default
+
+  # used for inclusions
+  allow_collection_params \
```

Comment:
this is no longer needed. https://github.com/blakechambers/matterhorn/pull/69/files#diff-1f778ea8ff9d56261674bf6ecde3f6c0R23